src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+// Actions
+const checkStatus = 'CHECK_STATUS';
+
```

Comment:
- [ ] It would be nice if the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) :blush: which is a requirement of this project, so it will be something like this :arrow_down: 
```javascript
const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'
```